src/actinia_stac_plugin/__init__.py

```diff
@@ -3,7 +3,7 @@
 
 try:
     # Change here if project is renamed and does not equal the package name
-    dist_name = "actinia_stac_plugin.wsgi"
+    dist_name = __name__
```

Comment:
is this change (and all following renaming/movement of files) intentional?

Reply:
Yes it is, it adjusts to the other plugins which are build via pyproject.toml.